### PR TITLE
Fix: Header Handle - handles don't show up [ED-1628]

### PR DESCRIPTION
**Pull request by guyc-E**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Comment by guyc-E** [comment]
I also fixed the handle of a section. ( check video in jira task comments )

File: assets/dev/scss/editor/preview/preview.scss [hunk 1]
```diff
@@ -108,6 +108,8 @@ html.elementor-html {
 }
 
 .elementor-section-wrap {
+	position: relative;
+	z-index: 1;
```
Comment (kobizz) [inline assets/dev/scss/editor/preview/preview.scss:112]: Add a comment that explains this index is needed

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/preview/preview.scss [hunk 2]
```diff
@@ -108,6 +108,8 @@ html.elementor-html {
 }
 
 .elementor-section-wrap {
+	position: relative;
+	z-index: 1; // Fix: Handle-Setting disappear when mouse hover the handle-setting in a section beneath the header.
```
Comment (kobizz) [inline assets/dev/scss/editor/preview/preview.scss:112]: It only happens if the header has a high z-index. You should mention it.

**Review by kobizz — CHANGES_REQUESTED** [review]

**Comment by kobizz** [comment]
@guyc-E Screenshotter is failing

**Pull request merged**